test/java/org/opendatakit/briefcase/transfer/TransferFormsTest.java

```diff
@@ -42,4 +62,27 @@ public void forms_from_varargs_factory_can_be_cleared() {
     forms.clear();
     assertThat(forms.isEmpty(), is(true));
   }
+
+  @Test
+  public void loads_all_forms() {
+    TransferForms transferForms = TransferForms.empty();
+    transferForms.load(forms);
+    assertEquals(10, transferForms.size());
+  }
+
+  @Test
+  public void test_select_all() {
+    TransferForms transferForms = TransferForms.from(forms);
+    transferForms.selectAll();
+    assertTrue(transferForms.allSelected());
+    assertTrue(transferForms.someSelected());
+  }
+
+  @Test
+  public void get_selected_forms() {
+    TransferForms transferForms = TransferForms.from(forms);
+    transferForms.selectAll();
+    transferForms.getSelectedForms();
+    assertEquals(10, transferForms.size());
```

Comment:
```suggestion
    TransferForms selectedForms = transferForms.getSelectedForms();
    assertEquals(10, selectedForms.size());
```
This test wasn't doing what you wanted because you weren't using the output of `getSelectedForms()` in the assertion.